### ISSUE TITLE
Fails to parse python2 print command with tuple list argument

### ISSUE THREAD

**Issue by ievans** [semgrep/semgrep#578]
**Describe the bug**
Sgrep fails to parse valid Python2 code, even with a python2 lang flag.

**To Reproduce**
Steps to reproduce the behavior, ideally a link to https://sgrep.live:

Given input:
```
#!/usr/bin/python

	print pkg + (package_length + 5 - len(pkg)) * ' ' + staging_version + (staging_length + 5 - len(staging_version)) * ' ' + release_version + (release_length + 5 - len(release_version)) * ' ',

	if staging_version != release_version:
		print '<------'
	else:
		print
````

```
sgrep -lang python2 -e '$X' ./tools/apt-checker/compare_packages.py
./tools/apt-checker/compare_packages.py:134:191: Syntax error

Fatal error: exception Parse_info.Parsing_error(_)
```

**Expected behavior**
This is valid python2 ,should parse

**Screenshots**

**Environment**
If not using sgrep.live: are you running off docker, an official binary, a local build?

**Comment by DrewDennison** [comment]
@aryx is this a good first issue?

**Comment by aryx** [comment]
yes

**Comment by nbrahms** [comment]
Fixed by https://github.com/returntocorp/pfff/pull/132

**Comment by ievans** [comment]
Confirmed fixed in 0.9.0 or earlier https://semgrep.live/pKlk
